src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -140,8 +140,9 @@ export default {
         // if we find some text ending with ", " and the next token is the start
         // of a new param, update this token text to replace the space with a
         // newline followed by 4 spaces
-        if (token.text && token.text.endsWith(', ')
-          && nextToken && nextToken.kind === TokenKind.externalParam) {
+        if (token.text && token.text.endsWith(', ') && nextToken
```

Comment:
@dobromir-hristov and @marinaaisa 

Would you find this more readable or less readable?

```diff
diff --git a/src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue b/src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue
index cf74b6cd..30e4d9cf 100644
--- a/src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue
+++ b/src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue
@@ -140,9 +140,9 @@ export default {
         // if we find some text ending with ", " and the next token is the start
         // of a new param, update this token text to replace the space with a
         // newline followed by 4 spaces
-        if (token.text && token.text.endsWith(', ') && nextToken
-          && (nextToken.kind === TokenKind.externalParam
-            || nextToken.kind === TokenKind.attribute)) {
+        const isStartOfParam = ({ kind }) => (kind === TokenKind.externalParam
+          || kind === TokenKind.attribute);
+        if (token.text && token.text.endsWith(', ') && nextToken && isStartOfParam(nextToken)) {
           newToken.text = `${token.text.trimEnd()}\n${indent}`;
           indentedParams = true;
         }
```

I kind of prefer it how it is, but I don't mind changing it based on your [feedback](https://github.com/mportiz08/swift-docc-render/pull/1#discussion_r1185824928) from the original PR

Reply:
Was gonna do something like this, but honestly I dont think we should spend much time over this :D 

```js
        const isStartOfParam = [TokenKind.externalParam, TokenKind.attribute].includes(nextToken.kind);
        if (token.text && token.text.endsWith(', ') && isStartOfParam)
```

Reply:
I think it's good to see the `isStartOfParam` on its own. It's a little easier to read since it clearly separates the 2 scenario. Maybe even better if we flip the order of the 2 scenarios in the comment, but not a big deal.

Reply:
> Was gonna do something like this, but honestly I dont think we should spend much time over this :D
> 
> ```js
>         const isStartOfParam = [TokenKind.externalParam, TokenKind.attribute].includes(nextToken.kind);
>         if (token.text && token.text.endsWith(', ') && isStartOfParam)
> ```

This would need to include a check that the next token exists first.

> I think it's good to see the `isStartOfParam` on its own. It's a little easier to read since it clearly separates the 2 scenario. Maybe even better if we flip the order of the 2 scenarios in the comment, but not a big deal.

I'll split it up since there's a consensus.

Reply:
I agree with @hqhhuang !

Reply:
6c594a2f5238e7eebc5ed9a94d08bca443e98267